.travis.yml

```
@@ -145,6 +144,33 @@ jobs:
       env:
         - CLANG_SANITIZER=TSAN
         - *common-job-env
+    - if: type != pull
+      name: snap build
```

"build" is redundant in a list of travis jobs.

```suggestion
      name: snap
```